modules/snooker/src/main/ts/ephox/snooker/api/TableOperations.ts

```diff
@@ -161,28 +161,39 @@ const opMakeRowHeader = (grid: Structs.RowCells[], detail: Structs.DetailExt, co
 };
 
 const opMakeRowsHeader = (initialGrid: Structs.RowCells[], details: Structs.DetailExt[], comparator: CompElm, genWrappers: GeneratorsTransform) => {
-  const rows = uniqueRows(details);
-
-  const replacer = (currentGrid: Structs.RowCells[], row: Structs.DetailExt) => TransformOperations.replaceRow(currentGrid, row.row, comparator, genWrappers.replaceOrInit);
+  const replacer = (currentGrid: Structs.RowCells[], row: Structs.DetailExt) =>
+    TransformOperations.replaceRow(currentGrid, row.row, comparator, genWrappers.replaceOrInit);
```

Comment:
Maybe replace `row.row` with `detail.row` (and `column.column` with `column.detail`) like you did below?! I know it was like that before but if we can do this little refactoring it'd be nice :)

Reply:
Not related to this change.

Reply:
Fixed